students/ArchWu/Session4/dict_lab.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python3
+
+def dict1():
+    """
+    Create a dictionary containing “name”, “city”, and “cake” for “Chris” from “Seattle” who likes “Chocolate” (so the keys should be: “name”, etc, and values: “Chris”, etc.)
+    Display the dictionary.
+    Delete the entry for “cake”.
+    Display the dictionary.
+    Add an entry for “fruit” with “Mango” and display the dictionary.
+    Display the dictionary values.
+    Display whether or not “cake” is a key in the dictionary (i.e. False) (now).
+    Display whether or not “Mango” is a value in the dictionary (i.e. True).
+
+    """
+
+    print(record1)
+    for v in record1:
```

Comment:
You may want to consider using .pop or del here. Otherwise you have to loop through the entire dict to find what you are trying to delete.

Ex: dict.pop(key) 
del dict[key]